packages/mco/components/dr-status-popover/dr-status-popover.tsx

```diff
@@ -158,43 +164,35 @@ const SyncStatus: React.FC<{
   );
 };
 
-const HelpLink: React.FC<{
-  href: string;
-  t: TFunction;
-  message: string;
-}> = ({ href, t, message }) => {
-  return (
-    <Button
-      className="dr-status-help-link"
-      variant={ButtonVariant.link}
-      isInline
-      href={href}
-      target="_blank"
-      rel="noopener noreferrer"
-      aria-label={t('Open documentation')}
-    >
-      {message}
-    </Button>
-  );
+const HelpLink: React.FC<{ href: string; t: TFunction; message: string }> = ({
+  href,
+  t,
+  message,
+}) => {
+  return <ViewDocumentation doclink={href} text={t(message)} padding="0" />;
```

Comment:
```suggestion
  return <ViewDocumentation doclink={href} text={message} padding="0" />;
```
translation only for the direct text like t("some text") not for variable like t(message)